### DISCUSSION TITLE
能否 将课程表做成电子表格一样 或者说导入一个电子表格

### DISCUSSION THREAD

**Discussion by CrazyU-U** [Class-Widgets/Class-Widgets#25] [建议]
不需要编辑 直接在线编辑 只要时间到 那个范围就显示 接下来的就自动展示课程 
还能否 调整弹窗位置

**Comment by MKStoler4096** [comment]
> 不需要编辑 直接在线编辑 只要时间到 那个范围就显示 接下来的就自动展示课程

是什么意思呢？“在线编辑”指类似集控那种统一发放？能再描述的清晰些吗？
@CrazyU-U

**Reply by QYlyj** [reply]
I think what he means may be that he wants to be able to set it up immediately without having to edit the course schedule in advance, but I think this function is not widely needed by users, but has nothing to do with the course schedule and teaching AIDS

**Comment by MKStoler4096** [comment]
导入电子表格和集控隔壁 ClassIsland 就能做到。
还有，如果您觉得想要需求某种功能，可以开一个 issue。
